frontend/lib/home/home_page.dart

```diff
@@ -51,6 +52,10 @@ class _HomePageState extends State<HomePage> {
 
   @override
   Widget build(BuildContext context) {
+    SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle(
+      // ios
+      statusBarBrightness: Brightness.light,
+    ));
```

Comment:
Should this really be part of the build function? For me it looks more like something for a onInit method